asv_bench/benchmarks/io/csv.py

```diff
@@ -5,9 +5,9 @@
 import numpy as np
 
 from pandas import Categorical, DataFrame, date_range, read_csv, to_datetime
-import pandas.util.testing as tm
 
 from ..pandas_vb_common import BaseIO
+from .pandas_vb_common import tm
```

Comment:
These will need to be up a dir

```suggestion
from ..pandas_vb_common import tm
```